### ISSUE TITLE
VerticalContentAlignment="Center" has been set, but the SourceHanSansCN font is still not centered in all controls

### ISSUE THREAD

**Issue by susin3x** [AvaloniaUI/Avalonia#17696]
### Describe the bug

VerticalContentAlignment="Center" has been set, but the SourceHanSansCN font is still not centered in all controls
![Snipaste_2024-12-05_11-24-45](https://github.com/user-attachments/assets/ed902dbc-59ed-4e5a-b707-e77429fb2984)
[Uploading FontFontFamilyTest.zip…]()

### To Reproduce

VerticalContentAlignment="Center" has been set, but the SourceHanSansCN font is still not centered in all controls

### Avalonia version

11.1.0

**Comment by timunie** [comment]
Please use 11.2 for testings or even better latest nightly. Also worth to check visual tree in DevTools F12 is there is any unexpected Margin or Padding.

**Comment by susin3x** [comment]
> Please use 11.2 for testings or even better latest nightly. Also worth to check visual tree in DevTools F12 is there is any unexpected Margin or Padding.
The SourceHanSansCN font will increase the Height of AccessText in the Button template
![Snipaste_2024-12-06_08-47-42](https://github.com/user-attachments/assets/a4736076-d5ee-438d-9d64-b0de409898ab)

**Comment by Hzlin7** [comment]
你用的字体可能不是最新的，可以去这下[Region Specific Subset OTFs Simplified Chinese (简体中文)](https://github.com/adobe-fonts/source-han-sans/releases/download/2.004R/SourceHanSansCN.zip)，或者使用SourceHanSansSC[Language Specific OTFs Simplified Chinese (简体中文)](https://github.com/adobe-fonts/source-han-sans/releases/download/2.004R/SourceHanSansSC.zip)

**Comment by timunie** [comment]
Hi @Hzlin7 so you think that the font is too old and newer font works fine? If so, I think we can move this topic into a Q&A as it is not an Avalonia issue then.

**Comment by Hzlin7** [comment]
> Hi @Hzlin7 so you think that the font is too old and newer font works fine? If so, I think we can move this topic into a Q&A as it is not an Avalonia issue then.

Yes, I am using the V2.004 version of the font, and it works fine.
